docker-compose-net/docker-compose-net.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: guides
+title: How to Use Docker Compose with Weave Net
+---
+
+Docker Compose is used to configure multiple container apps. A Docker Compose file configures your application's services and with a single command creates and starts services. Depending on how you are using Weave Net, whether through the Docker API Proxy or by using the Weave Docker Plugin, the syntax of the Docker Compose file differs. This guide provides an overview of the different ways in which networking can be configured and also provides examples of the Docker Compose files for Weave Net. 
```

Comment:
"and with a single command creates and starts services" => "and with a single command `docker-compose up` creates and starts services"